source/docs/software/advanced-controls/introduction/introduction-to-pid.rst

```diff
@@ -157,3 +157,9 @@ An *underdamped* response oscillates around the :term:`reference` before settlin
    :align: center
 
 is slow to rise and does not overshoot the :term:`reference`. A *critically damped* response has the fastest :term:`rise time` without overshooting the :term:`reference`.
+
+Other Information
+------------------
+
+* https://file.tavsys.net/control/controls-engineering-in-frc.pdf - See section 2 and section 6.
+* `VIDEO: What Is Feedforward Control? | Control Systems in Practice <https://www.youtube.com/watch?v=FW_ay7K4jPE>`_
```

Comment:
This video's explanation of feedforward is good, but the middle third brings up frequency domain stuff that isn't relevant to FRC students (they use modern controls for basically everything, whether they know it or not), and the last third presents a Simulink example that isn't reproducible by FRC students, cuz MATLAB costs money. frc-docs already has some JS tutorials for feedforward and feedback.